<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_filtr_GPS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>----</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_Lat">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_latitude">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_Lon">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lon:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_longitude">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title_accelerometer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accelerometer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_x_accelerometer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_x_accelerometer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_y_accelerometer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_y_accelerometer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_z_accelerometer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_z_accelerometer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title_accelerometer_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gyroscope</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_y_gyroscope">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_z_gyroscope">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_x_gyroscope">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_z_gyroscope">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_y_gyroscope">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_x_gyroscope">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_calibrate_sensors">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_send_read_command">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OnOff Relay</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_slave_14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slave 0x14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_slave_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NaN</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bluetooth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_btserver_on_off">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="CustomLabel" name="lb_draw_track">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>lb_draw_track</zorder>
   <zorder>bt_filtr_GPS</zorder>
   <zorder>lb_name_Lat</zorder>
   <zorder>lb_latitude</zorder>
   <zorder>lb_name_Lon</zorder>
   <zorder>lb_longitude</zorder>
   <zorder>lb_title_accelerometer</zorder>
   <zorder>lb_name_x_accelerometer</zorder>
   <zorder>lb_x_accelerometer</zorder>
   <zorder>lb_y_accelerometer</zorder>
   <zorder>lb_name_y_accelerometer</zorder>
   <zorder>lb_z_accelerometer</zorder>
   <zorder>lb_name_z_accelerometer</zorder>
   <zorder>lb_title_accelerometer_2</zorder>
   <zorder>lb_y_gyroscope</zorder>
   <zorder>lb_name_z_gyroscope</zorder>
   <zorder>lb_x_gyroscope</zorder>
   <zorder>lb_z_gyroscope</zorder>
   <zorder>lb_name_y_gyroscope</zorder>
   <zorder>lb_name_x_gyroscope</zorder>
   <zorder>bt_calibrate_sensors</zorder>
   <zorder>bt_send_read_command</zorder>
   <zorder>lb_name_slave_14</zorder>
   <zorder>lb_slave_14</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>label</zorder>
   <zorder>lb_btserver_on_off</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCurrent_Data">
    <property name="title">
     <string>Current Data</string>
    </property>
   </widget>
   <addaction name="menuCurrent_Data"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
